tools/fastlane-plugin/lib/fastlane/plugin/bugsnag/actions/send_build_to_bugsnag.rb

```diff
@@ -38,6 +38,9 @@ def self.run(params)
         payload[:sourceControl][:repository] = params[:repository] if params[:repository]
         payload[:sourceControl][:provider] = params[:provider] if params[:provider]
 
+        # If provided apply metadata to payload.
+        payload[:metadata] = params[:metadata]
+
```

Comment:
Added so that I could test metadata as other PR not merged yet. Will remove before merging to avoid clashes.